tests/test_processing.py

```diff
@@ -1106,671 +1158,600 @@ def test_run_disordered_tracing(
                         "confidence": np.float64(0.0),
                     }
                 },
-                3: {  # Grain
-                    1: {  # Node
-                        "error": False,
-                        "unmatched_branch_stats": {
-                            0: {"angles": np.float64(0.0)},
-                            1: {"angles": np.float64(49.50796095624354)},
-                            2: {"angles": np.float64(139.25543479929738)},
-                        },
-                        "node_coords": np.array(
-                            [[54, 70], [55, 69], [56, 68], [57, 66], [57, 67], [58, 66], [59, 66], [60, 65]]
-                        ),
-                        "confidence": None,
-                    }
-                },
             },
+            None,
             id="minicircle small",
+            marks=pytest.mark.skip(reason="disable whilst testing other params"),
         ),
         pytest.param(
-            "catenane_topostats",
+            "catenanes_post_disordered_tracing",
+            [0, 1],
             {
                 0: {  # Grain
                     1: {  # Node
                         "error": False,
                         "pixel_to_nm_scaling": 1,
                         "unmatched_branch_stats": {
                             0: {"angles": np.float64(0.0)},
-                            1: {"angles": np.float64(63.62466925374735)},
-                            2: {"angles": np.float64(29.510196308405636)},
-                            3: {"angles": np.float64(106.0045892179681)},
+                            1: {"angles": np.float64(139.2976869956517)},
+                            2: {"angles": np.float64(40.75146951858621)},
+                            3: {"angles": np.float64(116.78086102349846)},
```

Comment:
I assume that the apparent change in values of this result is actually okay and not due to a change in behavior?